static/js/encryption.js

```
@@ -0,0 +1,115 @@
+      throw new Error("Failed to encrypt message");
+    }
+  }
+
+  /**
+   * Decrypts a message
+   * @param {string} encryptedBase64 - Base64-encoded encrypted message
+   * @param {string} keyBase64 - Base64-encoded key
+   * @returns {Promise<string>} Decrypted message
+   */
+  static async decryptMessage(encryptedBase64, keyBase64) {
+    const key = await this.importKey(keyBase64);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)